apis/notificationserviceapi/notificationserviceapi.yaml

```diff
@@ -94,27 +96,21 @@ paths:
                           name: John
                           type: User
                         additionalInfo:
-                          group:
-                            id: '123434'
-                            name: Test
-                          groupRole: admin
-                          activity:
-                            id: do_12443
-                            type: Course
-                            name: Math's Activity
+                          
               c:
                summary: "Send SMS Notification"
                value:
                 request:
                   notifications:
                     - ids:
-                        - '12345'
+                        - '9824071234'
```

Comment:
9824071234 is not phone number?  we should not be leaking our phone number :)

Reply:
updated.